po/zh_CN.po

```diff
@@ -8,12 +8,104 @@
 #   - Zhuang Ya <zhuangya AT me.com>
 #   - Lian Cheng <rhythm.mail AT gmail.com>
 #
+#  Git glossary reference for Chinese translators
+#
+#   English                          |  Chinese
+#   ---------------------------------+--------------------------------------
+#   3-way merge                      |  三路合并
+#   alternate object database        |  备用对象库
+#   annotated tag                    |  附注标签
+#   bare repository                  |  纯仓库
+#   bisect                           |  二分查找
+#   blob object                      |  数据对象
+#   branch                           |  分支
+#   bundle                           |  包
+#   bypass                           |  绕过
+#   cache                            |  索引（的别称）
+#   chain                            |  （提交）链
+#   changeset                        |  变更集
+#   checkout                         |  检出
+#   cherry-picking                   |  拣选
+#   clean                            |  干净（的工作区）
+#   commit                           |  提交
+#   commit object                    |  提交对象
+#   commit-ish (also committish)     |  提交号
+#   core Git                         |  核心 Git 工具
+#   DAG                              |  有向无环图
+#   dangling object                  |  摇摆对象
+#   detached HEAD                    |  分离头指针
+#   directory                        |  目录
+#   dirty                            |  脏（的工作区）
+#   evil merge                       |  坏合并（合并引入了父提交没有的修改）
+#   fast-forward                     |  快进
+#   fetch                            |  获取
+#   file system                      |  文件系统
+#   Git archive                      |  仓库（对于 arch 用户）
+#   gitfile                          |  gitfile（仓库链接文件）
+#   grafts                           |  （提交）嫁接
+#   hash                             |  哈希值
+#   head                             |  头、分支
+#   HEAD                             |  头指针（当前分支）
+#   head ref                         |  分支
+#   hook                             |  钩子
+#   hunk                             |  补丁片段
+#   index                            |  索引
+#   index entry                      |  索引条目
+#   master                           |  master（缺省分支名）
+#   merge                            |  合并
+#   object                           |  对象
+#   object database                  |  对象库
+#   object identifier                |  对象标识符
+#   object name                      |  对象名称
+#   object type                      |  对象类型
+#   octopus                          |  章鱼式合并（两分支以上的合并）
+#   origin                           |  origin（缺省的远程名称）
+#   pack                             |  包
+#   pack index                       |  包索引
+#   pathspec                         |  路径规格
+#   pattern                          |  模式
+#   parent                           |  parent
```

Comment:
Yes.  Things are little bit tricky here.

Beside the choosing of "父/母", here's another related issue which comes into my mind:

The manual of `git-commit-tree` told us:

> A commit object may have any number of parents.  With exactly one parent, it is an ordinary commit.  Having more than one parent makes the commit a merge between several lines of history.  Initial (root) commits have no parents.

When a commit has more than 3 logical (directly) predecessors, the term "parent" itself is not accurate - none of us has more than **two** parents!
In such a case, the translation work is somehow embarrassing - "父提交", "母提交", and even "双亲提交" - none of them is 100% correct :-(
Maybe "前任提交" is a better choice.

However, we may consider this in another angle.
The Git manual use the term "parent" in some loose way - it just stands for all the object(s) in the logical predecessor(s) list.
When taking this into consideration, I think we can safely translate the term "parent" as "父提交" - a Git object may have many logical predecessor, each of which is a "父提交".

Return to the issue of choosing "父提交"/"母提交": I personally prefer "父提交" because I think  it has more inheritability with the term "ancestor".  Hopes the human rights organization would not argues against this ;-)